CONTRIBUTING.md

```diff
@@ -12,3 +12,17 @@
 * Be carefull when changing dependencies
 * You don't have to assign person and milestone
 * Document new features in [Wiki](https://github.com/4finance/micro-infra-spring/wiki), especially new [configuration properties](https://github.com/4finance/micro-infra-spring/wiki/Configuration) after they are merged
+
+# Rebasing your pull request
+You can consider rebasing your pull request instead of merging it. Let's assume that you have
+a branch named cool-feature. Here are instructions needed to rebase your branch on top of master:
+
+        $ git pull
+        $ git checkout cool-feature
+        $ git rebase master
```

Comment:
After `git pull` the local master should be up-to-date, but rebasing with origin/master also could be fine.